frame/support/procedural/src/pallet/parse/storage.rs

```diff
@@ -432,15 +463,15 @@ fn process_named_generics(
 				hasher1: parsed
 					.remove("Hasher1")
 					.map(|binding| binding.ty)
-					.expect("checked above as mandatory generic"),
+					.unwrap_or(syn::Type::Verbatim(quote::quote! { Blake2_128Concat })),
```

Comment:
I think you can do this:

```suggestion
					.unwrap_or(syn::parse_quote! { Blake2_128Concat }),
```